Tests/APITesters/AllAPITests/SwiftAPITester/SubscriptionInfoAPI.swift

```diff
@@ -32,4 +32,5 @@ func checkSubscriptionInfoAPI() {
     let txId: String? = subscription.storeTransactionId
     let isActive: Bool = subscription.isActive
     let willRenew: Bool = subscription.willRenew
+    let displayName = subscription.displayName
```

Comment:
We should specify the type to make the API tests correctly
```suggestion
    let displayName: String? = subscription.displayName
```